packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -4437,6 +4471,18 @@ class SemanticsConfiguration {
     _setFlag(SemanticsFlag.isHeader, value);
   }
 
+  /// Indicates the owning [RenderObject]'s hierarchy level.
```

Comment:
This doc is a bit confusing. what is a hierarchy level?

Reply:
Can you add more detail to this?

Reply:
Sure, I tried to add more details. Please let me know if still should add more information. Thank you.